CMakeLists.txt

```diff
@@ -252,6 +252,8 @@ if(NVFUSER_BUILD_WITH_ASAN)
   add_link_options(-fsanitize=address)
 endif()
 
+list(FILTER TORCH_LIBRARIES EXCLUDE REGEX "libcuda\.so")
```

Comment:
Future note: Why pytorch is adding the driver API as its libraries? Can we just not link with `TORCH_LIBRARIES`?

Reply:
interesting. I don't know if that's intentional, maybe it's a suggestion we can make to upstream folks.

In terms of what we link, I guess we can manually setup pytorch. But that seems like too big a hammer, since that's the suggested pytorch way to link against them?! https://pytorch.org/tutorials/advanced/cpp_frontend.html